src/test-runner/assertions.ts

```diff
@@ -0,0 +1,382 @@
+const chai = require("chai");
+import { ApiPromise, Keyring } from "@polkadot/api";
+const utilCrypto = require("@polkadot/util-crypto");
+import minimatch from "minimatch";
+import path from "path";
+import { BackchannelMap } from ".";
+import { DEFAULT_INDIVIDUAL_TEST_TIMEOUT } from "../constants";
+import {
+  chainCustomSectionUpgrade,
+  chainUpgradeFromLocalFile,
+  chainUpgradeFromUrl,
+  connect,
+  findPatternInSystemEventSubscription,
+  validateRuntimeCode,
+} from "../jsapi-helpers";
+import { Network } from "../network";
+import { FnArgs } from "../types";
+const { assert, expect } = chai;
```

Comment:
If you use the `import chai from "chai"` suggestion mentioned above then you can easily do this instead (removing the whole line here:
```suggestion
import { assert, expect } from "chai";
```